packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/add-notebook.html

```diff
@@ -0,0 +1,254 @@
+                <!-- step 5 - API -->
+                <h2
+                    id="API"
+                    data-translate="data_processing_add_notebook_stepper_api_title"
+                ></h2>
+
+                <oui-collapsible
+                    heading="{{:: 'data_processing_add_notebook_api_equivalent' | translate }}"
+                >
+                    <pci-order-command
+                        data-order-api-url="$ctrl.orderAPIUrl"
+                        data-order-data="$ctrl.orderData"
+                        data-help-link="$ctrl.addNotebookGuideUrl"
+                        data-auto-height="true"
+                        data-user="$ctrl.user"
+                    >
+                        <p
+                            data-translate="data_processing_add_notebook_command_description"
+                        ></p>
+                        <a
+                            class="oui-link_icon"
+                            target="_blank"
+                            rel="noopener"
+                            data-ng-href="{{$ctrl.apiGuideUrl}}"
+                        >
+                            <span
+                                data-translate="data_processing_add_notebook_command_documentation_link"
+                            ></span>
+                            <span
+                                class="oui-icon oui-icon-external-link"
+                                aria-hidden="true"
+                            ></span>
+                        </a>
+                    </pci-order-command>
+                </oui-collapsible>
+            </form>
+        </div>
+        <div class="col-lg-3">
+            <div class="order-summary sticky-top mb-3">
+                <oui-tile
+                    data-heading="{{:: 'data_processing_add_notebook_spec_heading' | translate }}"
+                >
+                    <oui-tile-definition>
+                        <oui-tile-description>
+                            <!-- Nom -->
+                            <div class="mt-2">
+                                <a
+                                    href=""
+                                    data-ng-click="$ctrl.scrollTo('name')"
+                                    data-translate="data_processing_add_notebook_spec_name"
+                                ></a>
+                                <span data-ng-bind="$ctrl.state.name"></span>
+                            </div>
+                            <!-- notebook type -->
+                            <div class="d-flex align-items-center mt-2">
+                                <div class="mr-2">
+                                    <a
+                                        href=""
+                                        data-ng-click="$ctrl.scrollTo('engine')"
+                                        data-translate="data_processing_add_notebook_spec_engine"
+                                    ></a>
```

Comment:
Please use a button instead of an empty href link

Reply:
Same answer as https://github.com/ovh/manager/pull/8470#discussion_r1039651102